### ISSUE TITLE
Cleanup nexus snaphots

### ISSUE THREAD

**Issue by maybeec** [oasp/oasp4j#415]
The jenkins is currently not building anymore as the available storage on the host is too low. One major issue here is the nexus snapshot repository, which already consumes up to 16GB of storage. Is there anybody with admin rights, who can cleanup the snapshots respository a little bit?

**Comment by ivanderk** [comment]
I propose to train & grant privileges to members of the Devon team. They are 100% available for these kind of maintenance tasks

**Comment by sobkowiak** [comment]
I think, we can configure a trigger which will remove old snapshots and leave only n (=1) last snapshots.

**Comment by sobkowiak** [comment]
I have performed following changes in Nexus:

**oasp-ci**:
* The existing *Cleanup Snapshots* task moved the snapshots to trash. I have changed the option to remove snapshots immediately.
* I have started a task which purges the trash filled by the *Cleanup Snapshots* task
* The *Cleanup Snapshots* task removes now the released snapshots (having corresponding released artifacts) after 60 day grace period. We don't need snapshots for released versions

**devon-ci**:
* In this instance there was no *Cleanup Snapshots* task. I have configured it simillar to the oasp-ci instance
